plugin/trino-elasticsearch/src/main/java/io/trino/plugin/elasticsearch/ElasticsearchConfig.java

```diff
@@ -76,6 +76,13 @@
     private boolean ignorePublishAddress;
     private boolean verifyHostnames = true;
 
+    private boolean unionSchemaIndicesForAlias;
+    private boolean failOnAliasSchemaMismatch = true;
```

Comment:
@findepi any thoughts here?

Reply:
Makes sense. Fewer configurations to provide when trying out this connector.

Reply:
I just remembered [Martin's comment here](https://github.com/trinodb/trino/issues/2748#issuecomment-583075591) that we should verify that this is what Elasticsearch does in this case.

Reply:
Few observations in ES for type mismatch:

- When selecting the field having type mismatch, it returns the document with actual datatype in its respective document. No error here because ES doesn't check the datatype validation.
- When filtering on the field having type mismatch, it fails to query if unable to cast to its actual datatype in any of the indices.

Respective observations in Presto with this fix:

- When selecting the field having type mismatch, presto returns the row with `default-datatype-for-alias-schema-mismatch` datatype if the casting is successful. Else fails with presto error.
- When filtering on the field having type mismatch, presto fails to query if unable to cast to `default-datatype-for-alias-schema-mismatch` datatype.

Both scenarios seem ok to me. 

Thoughts?

Reply:
`failOnAliasSchemaMismatch` default value is set to `false`. Closing this as of now. Please review and reopen again.